handlers/bin/command-not-found

```diff
@@ -73,57 +78,48 @@ def find_package_by_file(term):
     return None
 
 
-try:
-    if len(sys.argv) < 2:
-        sys.exit(EX_NOTFOUND)
-
-    import scout
-
-    default_lang = scout.DefaultLang(textdomain='command-not-found')
-    default_lang.install()
-
-    term = sys.argv[1]
-    print('%s:' % term, _('searching ...'), end=' ', file=sys.stderr)
-    lendel = len(term) + 3 + len(_('searching ...'))
-
-    import rpm
-
-    for path in ('/usr/sbin', '/sbin'):
-        pkg = find_package_by_file(path + '/' + term)
-        if pkg:
-            print('\r', lendel * ' ', end=' ', file=sys.stderr)
-            print_installed(term, pkg, path)
+if __name__ == '__main__':
+    try:
```

Comment:
very nice, can I suggest to create a main() function that contains the code and optionally throws an exception, and do the try here?